commit 35408908a673084710e562e294aa30a6ead4d389
Author: Arnaud Chong <achong@ocs.online.net>
Date:   Thu Sep 19 15:32:38 2013 +0000

    fix skonf.cfg

Index: shinken-1.4/etc/skonf.cfg
===================================================================
--- shinken-1.4.orig/etc/skonf.cfg	2013-09-19 15:55:52.000000000 +0000
+++ shinken-1.4/etc/skonf.cfg	2013-09-19 15:56:07.000000000 +0000
@@ -1,5 +1,5 @@
 # Log file of the discovery command
-log_file=/tmp/skonf.log
+log_file=/var/log/shinken/skonf.log
 
 # Configuration files with common discovery objects
 # like discoveryrules or runners
@@ -42,8 +42,8 @@
 
 # User that will be used by the arbiter. 
 # If commented, run as current user (root?)
-shinken_user=shinken
-shinken_group=shinken
+shinken_user=nagios
+shinken_group=nagios
 
 # Strip FQDN of the name ID to keep only the
 # basename of the element
